cfg4k-core/build.gradle

```diff
@@ -8,5 +8,6 @@ dependencies {
 
     testCompile libraries.junitrunner
     testCompile libraries.expekt
+    testCompile libraries.mockwebserver
```

Comment:
Not sure how you feel about pulling in more dependencies, but I assumed that a test dependency is fine if it makes tests easier and more readable.

Reply:
Tests dependencies are not a problem.